docs/user-guide/release-notes/2.9.x.rst

```diff
@@ -23,3 +23,25 @@ New Features
 * The yum distributor now supports the optional parameter
   ``packages_directory`` which can be used for custom destination directory
   for packages during the publish process.
+
+Known Issues
+------------
+
+Several issues were reported against Pulp 2.8 that were not included in the Pulp 2.9.0 release
+as a result of release timing. The list of :fixedbugs:`2.8.6` outlines these bugs, but
+there is one issue in particular that can potentially break RPM repository syncing after
+upgrading: https://pulp.plan.io/issues/2048
```

Comment:
You could use the sphinx command to shorten the URL.  http://docs.pulpproject.org/dev-guide/contributing/documenting.html#editing-the-docs